server/src/main/java/org/opentosca/toscana/plugins/cloudfoundry/FileCreator.java

```diff
@@ -222,6 +233,6 @@ private void addProviderServiceOfferings(BashScript deployScript) throws IOExcep
             }
             deployScript.append(String.format("# %-20s %-40s %-50s ", service.getLabel(), plans, service.getDescription()));
         }
-        deployScript.append("\n* These service plans have an associated cost. Creating a service instance will incur this cost.");
+        deployScript.append("\n# * These service plans have an associated cost. Creating a service instance will incur this cost.");
```

Comment:
Shouldn't this just be `# These service...` instead of `\n# * These service...`

Reply:
No, I need an additional new line and the `*` is also needed as annotation.